twine/utils.py

```diff
@@ -201,9 +204,9 @@ def check_status_code(response: requests.Response, verbose: bool) -> None:
     except requests.HTTPError as err:
         if response.text:
             if verbose:
-                print("Content received from server:\n{}".format(response.text))
+                logger.info("Content received from server:\n{}".format(response.text))
             else:
-                print("NOTE: Try --verbose to see response content.")
+                logger.warning("NOTE: Try --verbose to see response content.")
```

Comment:
Same note as above re: not checking `verbose` and letting `logging` do the work.

```python
            logger.info("Content received from server:\n{}".format(response.text))
            logger.warning("NOTE: Try --verbose to see response content.")
```

However, this makes the `verbose` argument to the function unnecessary, and I'm not sure how to deal with that.

Reply:
If we do this then the "NOTE: Try --verbose to see response content" will be printed even if the user is using the `--verbose` option as a `warning` level log will be logged even if the log level is set to `info`. I think leaving `verbose` as a parameter for the function makes sense here as it would allow for selectively printing the "NOTE: Try --verbose to see response content" message without having to print conditionally on log level(which may be confusing to read) but that would also work.